scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -419,3 +419,48 @@ task CountSuperpartitions {
         Int num_superpartitions = read_int('num_superpartitions.txt')
     }
 }
+
+task ValidateFilterSetName {
+    input {
+        Boolean go = true
+        String filter_set_name
+        String data_project
+        String data_dataset
+        String query_project = data_project
+        String filter_set_info_timestamp = ""
+    }
+    meta {
+        # Not `volatile: true` since there shouldn't be a need to re-run this if there has already been a successful execution.
+    }
+
+    # add labels for DSP Cloud Cost Control Labeling and Reporting
+    String bq_labels = "--label service:gvs --label team:variants --label managedby:gvs_utils"
+
+    command <<<
+        set -o errexit -o nounset -o xtrace -o pipefail
+
+        echo "project_id = ~{query_project}" > ~/.bigqueryrc
+
+        OUTPUT=$(bq --location=US --project_id=~{query_project} --format=csv query --use_legacy_sql=false ~{bq_labels} "SELECT filter_set_name as available_filter_set_names FROM \`~{data_project}.~{data_dataset}.filter_set_info\` GROUP BY filter_set_name")
```

Comment:
Mind seeing if this works without the "--location=US" part and, if so, removing it?  As is, it'll fail if we specify a dataset anywhere else.  I'm 99.99% sure this call will function perfectly without that parameter.